assets/js/components/setup/SetupUsingProxyWithSignIn.js

```diff
@@ -180,6 +184,7 @@ export default function SetupUsingProxyWithSignIn() {
 
 	let title;
 	let description;
+	let learnMore;
```

Comment:
Let's call this something like `showLearnMoreLink` and initialise it to `false`, just to be a bit more clear/explicit and what it's doing.

Reply:
Updated as well, thank you!